### ISSUE TITLE
atom-text-editor.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by carolfalardeau** [forgecrafted/rulerz#20]
The contents of `atom-text-editor` elements are no longer encapsulated
within a shadow DOM boundary. Please, stop using `shadowRoot` and access
the editor contents directly instead.
```
atom-text-editor.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor-element.js:68:18)
ruler-view.getEditor (/Users/noovo/.atom/packages/rulerz/lib/ruler-view.coffee:38:33)
ruler-view.insert (/Users/noovo/.atom/packages/rulerz/lib/ruler-view.coffee:45:14)
ruler-view.initialize (/Users/noovo/.atom/packages/rulerz/lib/ruler-view.coffee:30:12)
createView (/Users/noovo/.atom/packages/rulerz/lib/ruler-manager.coffee:25:32)
ViewRegistry.createView (/Applications/Atom.app/Contents/Resources/app.asar/src/view-registry.js:117:78)
```

**Comment by sergeysova** [comment]
@BattleBrisket @cmtonkinson Please, fix it

**Comment by BattleBrisket** [comment]
See comment on #21

**Comment by cmtonkinson** [comment]
Thanks for using rulerz, and caring enough to submit a bug report! We're no longer actively using Atom day-to-day, and so haven't devoted a great deal of time to the package. If you're interested in picking up maintainership, please let me know.

**Comment by chmac** [comment]
Hopefully this was fixed in #22. Closing.
